examples/building_and_installing_viame/README.rst

```diff
@@ -50,13 +50,6 @@ a starting point.
 .. code-block:: bash
 
    sudo apt-get install git zlib1g-dev libcurl4-openssl-dev libexpat1-dev dh-autoreconf liblapack-dev libxt-dev
-   sudo apt-get build-dep libboost-all-dev qt5-default
-
-If using VIAME_ENABLE VIVIA for building GUIs, you also may need to run:
-
-.. code-block:: bash
-
-  sudo apt-get build-dep qt5-default
```

Comment:
Are these not necessary from a relatively fresh default Ubuntu 16.04?

Reply:
Sounded from our discussion the other day like you just wanted the references to build-dep gone. I can add the first back in. The second should stay gone, though, since they both have qt5-default. Having the second is redundant and repetitive.

Reply:
Can you reformulate the requirements to not use build-dep, just sudo apt-get install, whatever they are?